docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -192,6 +193,37 @@ test(`Test Speed`, async t => {
 });
 ```
 
+### Setting Page Load Timeout
+
+Page load timeout defines the amount of time passed after the `DOMContentLoaded` event, within which the `window.load` event should be raised.
+
+After the timeout passes or the `window.load` event is raised (whatever happens first), TestCafe starts the test.
+
+To specify the page load timeout in test code, use the `t.setPageLoadTimeout` method.
+
+```text
+t.setPageLoadTimeout( duration )
+```
+
+Parameter  | Type      | Description
+---------- | --------- | -----------
+`duration` | Number    | Page load timeout, in milliseconds. `0` to skip waiting for the `window.load` event.
```

Comment:
Should we mention default timeout value here?

Reply:
Not, because there is no default value for `t.setPageLoadTimeout`. Default value is actual for the `page-load-timeout` setting for the whole test run